### ISSUE TITLE
Styling Tweaks

### ISSUE THREAD

**Issue by TheEibwen** [all-rit/AccessibilityLab1#7]
The styling is a little lacking in the first accessibility tweaks, so the plans are:

- [x] Button hover/active styling
- [x] Gray out the sound option during the game
- Etc
